packages/polyserve/src/args.ts

```diff
@@ -91,8 +91,9 @@ export const args: ArgDescriptor[] = [
   },
   {
     name: 'npm',
-    description: 'Sets npm mode: component directory is "node_modules" and ' +
-        'the package name is read from package.json',
+    description: 'Sets npm mode: dependencies are installed from npm, ' +
+        'component directory is "node_modules" and the package name is read ' +
+        'from package.json',
```

Comment:
I'm starting to think that the `--npm` flag / config option description should be made more generic because it has to match across all commands ("Sets npm mode.") and each command's help description might be a better place to put specific information about how it reacts to the flag.

Reply:
Yeah, I think we need to do some unification to the way we handle options in the CLI